### ISSUE TITLE
ember scrollable uncaught TypeError

### ISSUE THREAD

**Issue by BennyAlex** [offirgolan/ember-light-table#262]
When I use ember light table, Ig get the following error:
ember-scrollable.js:210  Uncaught TypeError: Cannot read property 'scrollTo' of undefined(…)

**Comment by buschtoens** [comment]
This is probably a problem with [`alphasights/ember-scrollable`](https://github.com/alphasights/ember-scrollable).

Can you please give us an example of what exactly triggers this error?

**Comment by buschtoens** [comment]
Also, when you click the `ember-scrollable.js:210` part of the error message, the dev tools should show the line that is throwing the error. Can you copy this line and a few lines before and after please? :)

**Comment by BennyAlex** [comment]
It seems that I had to change some codes according to the new light-table version. Now it works

**Comment by BennyAlex** [comment]
I found out when using a height for the light-table and set t.head fixed=true the error will occur, e.g:

```
{{#light-table table height='30vh' as |t|}}
  {{t.head
  onColumnClick=(action 'onColumnClick')
  fixed=true
  }}
  {{#t.body
  multiSelect=true
  as |body|}}
    {{#if isLoading}}
      {{#body.loader}}
        loading...
      {{/body.loader}}
    {{else}}
      {{#if table.isEmpty}}
        {{#body.no-data}}
          no data
        {{/body.no-data}}
      {{/if}}
    {{/if}}
  {{/t.body}}
{{/light-table}}
```

**Comment by buschtoens** [comment]
Setting `fixed=true` enables ember-scrollable in the first place, so this is not the origin of the error. 

Are you sure everything's up to date? Maybe delete `node_modules` and then reinstall with `npm i`. 

If the errors persists, please post the relevant code of `ember-scrollable.js`.

**Comment by BennyAlex** [comment]
no, fixed=true is for the fixed table head, that I will be visible on scroll. 
Can you try it? I dont have a blank ember installation

**Comment by buschtoens** [comment]
We cannot reproduce the error. You must be doing something differently. 

I know that `fixed=true` sets the table header or footer to be fixed. And having a fixed header or footer is what enables ember-scrollable in the first place. 

Again, please post the relevant code from `ember-scrollable.js`. A full stack trace would be nice as well.

**Comment by BennyAlex** [comment]
hm I think that light-table isnt compatible with ember-paper

**Comment by buschtoens** [comment]
Could be. I cannot verify this right now as I'm on vacation over the weekend. 🏂

In the meantime you could post the relevant code and stack trace.

**Comment by angelomachado** [comment]
Maybe related to [#27](https://github.com/alphasights/ember-scrollable/issues/27)

**Comment by buschtoens** [comment]
@angelomachado Most likely not related.

The error message is: 
`ember-scrollable.js:210 Uncaught TypeError: Cannot read property `**`'scrollTo'`**` of undefined(…)`

It would have to contain `parseInt` or `isNaN` instead.

**Comment by buschtoens** [comment]
I'm really starting to suspect, that @BennyAlex is using an old release (0.3.4) of ember-scrollable, which is why I asked him to nombom.

> Are you sure everything's up to date? Maybe delete node_modules and then reinstall with npm i.

 There is no other way this error can occur.

[See for yourself](https://github.com/alphasights/ember-scrollable/commit/0d84e3f77526521b2d4a613c72a946c98ec9b02b#diff-499447ac5ce8a7c1077b7503f9403d0dL210).

Perhaps we can close this issue.

**Comment by buschtoens** [comment]
Our `package.json` still allowed for ember-scrollable@0.3.4. Fixed in #266.
